pinc/wordcheck_engine.inc

```diff
@@ -10,7 +10,7 @@ include_once($relPath.'misc.inc'); // str_contains, array_get
 // NOTE: This should probably be changed when moving to UTF-8
 //       at that time we can probably move to using \W instead of
 //       the mess below
-$word_letters = "a-zA-Z�-��-��-�";
```

Comment:
I suspect your editor saved the file as UTF-8 instead of ISO-8859-1. We actually need the file to stay the latter until the Unicode branch merges (the file was already converted to UTF-8 in that branch).

Reply:
Fixed it by changing visual studio code to save in iso-8859-1 for now until unicode comes over.